.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,6 +1,6 @@
 <!--
 
-Please prefix the title with the platform you are targetting:
+Please prefix the title with the platform you are targeting:
```

Comment:
Laurent uses British English and therefore `targetting` is correct.

Reply:
Sorry, reverted in 7de5f7f

Reply:
Thanks.